### ISSUE TITLE
[FEATURE-REQUEST]: Automatically resize cape images (0.6.9-beta.25)

### ISSUE THREAD

**Issue by BigBrainRobin29** [NoRiskClient/issues#685]
### Your idea

Dass die Capes automatisch resized werden, am besten mit Notification "Your image has been automatically resized" oder so. Ist nur so ne Idee, weil auf dc so viele das nicht checken.

### Where would this be applied?

Launcher

### What platforms would this affect?

Windows, MacOS, Linux

**Comment by TimLohrer** [comment]
Nee, weil das je nach dem was da uploaded wird auch scheiße is. Die sollen einf template usen.

**Comment by BigBrainRobin29** [comment]
ich will das jetzt haben, vielleicht mit modal "dein cape hat die falsche res, willst du es automatisch resizen" oder so
es gibt viel zu viele tickets dazu

**Comment by copyandexecute** [comment]
Also clientsided check am besten wrong size? oder wirklich manuell resizen
